core/src/main/java/com/linecorp/armeria/server/SetPathMapping.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2015 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+public final class SetPathMapping extends AbstractPathMapping {
+
+    static final String PREFIX = "path set:";
+
+    private final Set<String> pathSet;
+    private final String pathSetStr;
+    private final String loggerName;
+    private final String strVal;
+
+    /**
+     * Constructs {@link SetPathMapping} with the path set it maps.
+     *
+     * @param pathSet the set of path.
+     */
+    public SetPathMapping(Set<String> pathSet) {
+        this.pathSet = pathSet;
+        pathSetStr = String.join(", ", pathSet);
+        loggerName = loggerName(pathSetStr);
+        strVal = PREFIX + pathSetStr;
+    }
+
+    @Override
+    protected PathMappingResult doApply(PathMappingContext ctx) {
+        return pathSet.contains(ctx.path()) ? PathMappingResult.of(ctx.path(), ctx.query())
+            : PathMappingResult.empty();
```

Comment:
Thanks for reminding. 

BTW, though current style check is good, while how about using google-java-format to help reduce format related time cost of engineers?
https://github.com/google/google-java-format
Both gradle and IntelliJ/Eclipse plugin are available for auto code reformating.